### ISSUE TITLE
Adding Power support for Cadvisor image

### ISSUE THREAD

**Issue by lysannef** [google/cadvisor#2287]
Hi All,

The images available here -  have support for "amd64" only as seen below

 docker inspect google/cadvisor:latest | grep Arch
 "Architecture": "amd64",

I was able to build this image locally on a Power machine using the [dockerfile](https://github.com/google/cadvisor/blob/master/deploy/Dockerfile) and validate it successfully and was looking for help in making the official image on dockerhub, multi-arch.

Thank you!!

**Comment by dashpole** [comment]
cc @bentheelder
Do we have support for mult-arch builds in k8s that we could use to solve this?

**Comment by BenTheElder** [comment]
I wouldn't necessarily re-use any of the k8s build tooling, but we do cross compile to many architectures including PPC64LE.

**Comment by lysannef** [comment]
@dashpole Have there been any discussions on taking this forward?

**Comment by dashpole** [comment]
I don't think we would push images we haven't built ourselves.  If there is a way to do a containerized multi-arch build (so we can build it without a power machine), we could consider adding that functionality to cAdvisor's build scripts, and making it part of the release.  I don't really have any bandwidth to tackle it, but I would be happy to review changes.

I can't guarantee the image will be pushed to dockerhub, as I seem to have lost access.  But it would still exist on k8s.gcr.io.

**Comment by lysannef** [comment]
@dashpole  Thanks for the updates!
We will work on getting a containerized multi-arch build and will raise a PR for the same.

**Comment by dims** [comment]
Closing this issue as it has been open since before January 1, 2024. If this is still relevant, please feel free to reopen or create a new issue.
